.circleci/config.yml

```diff
@@ -197,7 +197,7 @@ jobs:
                 root: ios/output/gym
     bump_version:
         docker:
-            - image: circleci/node:12.16.3
+            - image: circleci/node:14.16.0
```

Comment:
Is it possible these updates make the build a lot slower? Before the e.g. check-step takes aroung 30sec and now more than a few minutes.

Reply:
fixed after adding `--maxWorkers=1` to yarn test